src/app/commands/cmd_change_pixel_format.cpp

```diff
@@ -534,7 +534,9 @@ void ChangePixelFormatCommand::onLoadParams(const Params& params)
     else {
       render::DitheringMatrix ditMatrix;
       if (!load_dithering_matrix_from_sprite(matrix, ditMatrix))
-        throw std::runtime_error("Invalid matrix name");
+        throw std::runtime_error(
+          fmt::format("The dithering matrix file {} doesn't exist",
+                      matrix));
```

Comment:
Probably I'll move the throw `std::runtime_error` inside `load_dithering_matrix_from_sprite` to avoid duplicating this exact same code here and `Extension::DitheringMatrixInfo::matrix()`.